samples/web/content/apprtc/params.html

```diff
@@ -0,0 +1,97 @@
+
+  <base target="_blank">
+
+  <title>AppRTC parameters</title>
+
+  <link rel="icon" sizes="192x192" href="images/webrtc-icon-192x192.png">
+  <link href="//fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet" type="text/css">
+  <link rel="stylesheet" href="https://googlechrome.github.io/webrtc/samples/web/css/main.css" />
+
+  <style>
+  div#container {
+    max-width: 65em;
+  }
+    h2 {
+      font-size: 1.5em;
+      font-weight: 500;
+    }
+    h3 {
+      border-top: none;
+    }
+    section {
+      border-bottom: 1px solid #eee;
+      margin: 0 0 1.5em 0;
+      padding: 0 0 1.5em 0;
+    }
+    section:last-child {
+      border-bottom: none;
+      margin: 0;
+      padding: 0;
+    }
+    td {
+      padding: 0.5em 1em;
+      vertical-align: top;
+    }
+    td:first-of-type {
+      text-align: right;
+    }
+  </style>
+</head>
+<body>
+  <div id="container">
+
+    <h1>AppRTC parameters</h1>
+
+    <section>
+
+      <p>A number of settings for the <a href="https://apprtc.appspot.com">AppRTC video chat application</a> can be changed by adding URL parameters.</p>
+
+      <p>For example: <a href="https://apprtc.appspot.com/?hd=true&amp;stereo=true&amp;debug=loopback">https://apprtc.appspot.com/?hd=true&amp;stereo=true&amp;debug=loopback</a></p>
+
+      <p>The file using the parameters is <a href="https://github.com/GoogleChrome/webrtc/blob/master/samples/web/content/apprtc/apprtc.py" title="apprtc.py file in AppRTC repo">apprtc.py</a>. More Google-specific parameters are available from the <a href="//code.google.com/p/webrtc/source/browse/trunk/talk/app/webrtc/mediaconstraintsinterface.h" title="mediaconstraintsinterface.h">MediaConstraints interface</a>.</p>
+
+      <p>For more information see <a href="http://gingertech.net/2014/03/19/apprtc-googles-webrtc-test-app-and-its-parameters/">AppRTC : Google's WebRTC test app and its parameters</a>.</p>
+
+    </section>
+
+    <section>
+
+      <table>
+        <tr><td><a href="https://apprtc.appspot.com?hd=true">hd=true</a></td><td>Use HD camera resolution <a href="http://googlechrome.github.io/webrtc/samples/web/content/getusermedia/resolution/" title="Video constraints demo">constraints</a>, i.e. minWidth: 1280, minHeight: 720</td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?stereo=true">stereo=true</a></td><td>Turn on stereo audio</td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?debug=loopback">debug=loopback</a></td><td>Connect to yourself, e.g. to test firewalls</td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?ts=turn:foo">ts=[turnserver]</a></td><td>Set TURN server different from the default</td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?tp=foo">tp=[password]</a></td><td>Set password for the TURN server</td></tr>
+        <tr><td><a href="https://apprtc.appspot.com?audio=true&amp;video=false">audio=true&amp;video=false</a></td><td>Audio only</td></tr>
```

Comment:
I thought we were planning to generate the list of params through template. It's hard to keep them in sync this way. 
Also, why not put this in a separate CL?